aleph_alpha_client/image.py

```diff
@@ -9,30 +9,43 @@ class ImagePrompt:
     base64.
     """
 
-    def __init__(self, base_64: str):
+    def __init__(self, base_64: str, x: int = None, y: int = None, size: int = None):
         # We use a base_64 reperesentation, because we want to embed the image
         # into a prompt send in JSON.
         self.base_64 = base_64
+        self.x = x
+        self.y = y
+        self.size = size
 
     @classmethod
-    def from_url(cls, url: str):
+    def from_url(cls, url: str, x: int = None, y: int = None, size: int = None):

```

Comment:
`int = None` is always wrong. Shouldn't it be `Optional[int] = None`. Anyhow, rather than default arguments, I would prefer a `from_url_with_cropping` method there the cropping arguments are not optional. This way it is also implicitly clear, that either all or none of them have to be specified.